trailmet/algorithms/binarize/birealnet_model.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+"""BiRealNet.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1oH01ykDQAnsmSdTEoGAjmkRw5NuxeIVV
+"""
+
+import torch
+import torch.nn as nn
+import torch.utils.model_zoo as model_zoo
+import torch.nn.functional as F
+import numpy as np
+
+__all__ = ['birealnet18', 'birealnet34']
+
+def conv3x3(in_planes, out_planes, stride=1):
```

Comment:
can you use this function from https://github.com/transmuteAI/trailmet/blob/dev/trailmet/models/resnet.py

Reply:
done